src/ripple/ledger/PaymentSandbox.h

```diff
@@ -76,6 +83,7 @@ class DeferredCredits
             Currency const& c);
 
     std::map<Key, Value> map_;
+    std::map<AccountID, std::uint32_t> ownerCounts_;
```

Comment:
The name `ownerCounts_` really describes what's here.  In contrast, `map_` is not as descriptive.  Can you think of a better name for `map_`?